exchanges/request-policy/src/requestPolicyExchange.ts

```diff
@@ -46,7 +46,14 @@ export const requestPolicyExchange = (options: Options): Exchange => ({
   };
 
   const processIncomingResults = (result: OperationResult): void => {
-    operations.set(result.operation.key, new Date().getTime());
+    const meta = result.operation.context.meta;
```

Comment:
Can meta be relied upon in non development settings? I ask because the comment above the meta in types says `Metadata that is only available in development for devtools.` 

https://github.com/FormidableLabs/urql/blob/d1a2adba7bbd7b513213b1a3118d17118456bd81/packages/core/src/types.ts#L34-L53

Reply:
you can rely on `cacheOutcome` the other three properties are devtools, we should maybe clarify that 😄

Reply:
👍 Sweet just wanted to make sure. Would be good to slightly tweak that comment at some point to avoid confusion